DASH-IF-Ingest.bs.md

```diff
@@ -545,38 +545,28 @@ DASH-IF makes no warranty whatsoever for such third party material.
 
 # General Ingest Protocol Behavior # {#general}
 
-  The media ingest follows the following  
-  general requirements for both targetted interfaces.  
-  The ingest interface uses existing media formats 
-  in combination with the HTTP POST method.
+  Live media ingest follows the following  
+  general requirements. It is recommended that 
+  a conforming implementation fullfills these requirements.
 
-     1. The [=Ingest source=]  SHALL communicate
-        using the HTTP POST method as defined in
+     1. The [=Ingest source=] SHALL communicate
+        using the HTTP POST or PUT method as defined in
         the HTTP protocol, version 1.1 [[!RFC7235]]
-     2. The [=Ingest source=]  SHOULD
+     2. The [=Ingest source=] SHOULD
```

Comment:
Actually, since the creation of this document, TLS 1.3 has been standardised under RFC 8446 - we can now update this language to say "The Ingest source SHOULD use TLS 1.2 at minimum".